OpenDreamRuntime/EntryPoint.cs

```diff
@@ -72,8 +72,8 @@ public sealed class EntryPoint : GameServer {
 
         protected override void Dispose(bool disposing) {
             // Write every savefile to disk
-            foreach (var savefile in DreamObjectSavefile.Savefiles) {
-                savefile.Flush();
+            foreach (var savefile in DreamObjectSavefile.Savefiles.ToArray()) { //ToArray() to avoid modifying the collection while iterating over it
```

Comment:
Copying the list seems unnecessary, but it's probably fine if it doesn't grow too big (especially once softdels are added)